### DISCUSSION TITLE
cannot find src/shared/static.json during deployment

### DISCUSSION THREAD

**Discussion by bearzk** [beginner-corp/begin-community#80] [Support] [answered]
my deployment failed with message
`ENOENT: no such file or directory, stat '/tmp/QB7H72Drvyv/src/shared/static.json'`

the codebase works locally, I suspect here the verify step wanted to make sure all files exist, but before the build step,
there's no `./dist` folder yet, then the `src/shared/static.json` is pointing to non-existing `./dist/static.json`.

any hint on how to fix this problem?

**Comment by ryanblock** [comment, marked as answer]
Hmm, haven't seen this one before, happy to take a look (and thank you for your patience!). Can you share:

- Repo (if public)
- If not public, the Arc project manifest (if possible)
- Begin app ID?

Thanks!

**Reply by bearzk** [reply]
thank you for responding!

now I've passed that step by setting `@static fingerprint false`, though I'm not sure if this is prefered for static assets. but still, I cannot see my site.

begin app id: `wr2gunwzgd7zwj`

current `.arc` file of the project
```
@app
it-burns

@static
fingerprint false
folder /dist

@http
post /api/messages
post /api/messages/:id
delete /api/messages/:id

@tables
data
  scopeID *String
  dataID **String
  ttl TTL
```

in the app dashboard, HTTP routes tab, I can see there's one `GET /*` route, and it's not defined in the `.arc` file.
it seems it caught all my requests to the site and failed with log messages like this:
`[ERROR] [1616582606100] LAMBDA_RUNTIME Failed to post handler success response. Http response code: 413.`

**Reply by brianleroux** [reply]
hey @bearzk can you add one of us as a collaborator on this repo? something is def funky w the code!

**Reply by ryanblock** [reply]
Not sure which code you're referring to but this sounds like a Begin problem to me?

**Reply by bearzk** [reply]
thank you very much! @brianleroux @ryanblock 

I invited you as collaborators in the repo.

**Reply by brianleroux** [reply]
nice I got it!!! looks like Vue (or webpack?) needs `NODE_ENV` to be  `production` to do its business so I added to your build script. 

cool app btw!

**Reply by bearzk** [reply]
@brianleroux thank you very much for your help!

you left a funny commit message there :)
